src/playground-project.ts

```diff
@@ -160,6 +160,30 @@ export class PlaygroundProject extends LitElement {
     ).href;
   }
 
+  private _filesSetExternally = false;
+
+  /**
+   * Get or set the array of project files.
+   *
+   * Files set through this property always take precedence over `projectSrc`
+   * and slotted children.
+   */
+  @property({type: Object})
```

Comment:
if you're not using the attribute, you don't really need the decorator, you could just call `requestUpdate` in the setter.

Reply:
Done.